deluca/envs/classic/_pendulum.py

```diff
@@ -46,14 +46,56 @@ def __init__(self, reward_fn=None, seed=0):
 
         self.state_size = 2
         self.action_size = 1
+        self.action_dim = 1 # redundant with action_size but needed by ILQR
+        
+        self.H = 50
 
         self.n, self.m = 2, 1
         self.angle_normalize = angle_normalize
+        self.nsamples = 0
 
         self.random = Random(seed)
 
         self.reset()
-
+        
+        @jax.jit
+        def _dynamics(state, action):
```

Comment:
Changing from method (i.e., `def dynamics(self, etc)`) to function attribute (i.e., what we have here) is probably ok for now, but after we make the jump to flax, we probably want to be consistent across environments. So nothing to do for now (so long as loop/scan work), but something to keep in mind!

Reply:
Actually I still have it as a method ultimately, if you see below it we have the line self.dynamics = _dynamics. I think the point of me moving the dynamics method into the init as _dynamics was for the self.f, self.f_x, and self.f_u

Reply:
Yeah, there's a difference between method and function attribute (i.e., `self.dynamics = _dynamics`), but I was saying that I don't know if there's an important enough difference for us now. Just mentioning that the two are different.